src/Catty/DataModel/UserDataContainer/UserDataContainer.swift

```diff
@@ -0,0 +1,233 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+@objc(UserDataContainer)
+@objcMembers class UserDataContainer: NSObject {
+    private var variables: SynchronizedArray<UserVariable>
```

Comment:
Sorry, I did not get it at first why you renamed this with an underscore. Now it makes sense. I agree with your previous solution. Having underscore is much better because it allows us to have a clear interface to the outside. please change it back to `_variables` and `_lists`. Thanks a lot!